pkg/kubernetes/mtls.go

```diff
@@ -107,6 +110,31 @@ func ExportTrustChain(outputDir string) error {
 	return nil
 }
 
+// Check and warn if cert expiry is less than `warningDaysForCertExpiry` days.
+func CheckForCertExpiry() {
+	expiry, err := Expiry()
+	// The intent is to warn for certificate expiry only when it can be fetched.
+	// Do not show any kind of errors with normal command flow.
+	if err == nil {
+		daysRemaining := int(expiry.Sub(time.Now().UTC()).Hours() / 24)
+		if daysRemaining < warningDaysForCertExpiry {
+			warningMessage := ""
+			if daysRemaining == 0 {
```

Comment:
Same nit for line 120 as above.
For if/else checks from line 122, we can try using a switch case here for readability.

Reply:
I have mixed feelings here. Normally switch cases are good if we compare with constants like.  I might be wrong though. Thoughts ?
```
n=5
switch n {
case 1:
case 2:
default:
}
```

Reply:
Sure @pravinpushkar, it's only a nit. Go does support conditional switch case though:
```go
switch {
case x < 0:
        // do something
case x > 0:
        // do something
case x == 0:
        // do something
}
```